target/linux/ath79/image/generic.mk

```diff
@@ -722,6 +722,22 @@ define Device/engenius_ews511ap
 endef
 TARGET_DEVICES += engenius_ews511ap
 
+define Device/engenius_emr3000
+  SOC := qca9558
+  DEVICE_VENDOR := EnGenius
+  DEVICE_MODEL := EMR3000
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct \
+	kmod-usb2 kmod-usb-storage
```

Comment:
usb-storage is commonly not added as default package.

Reply:
Thanks for comment. OK, I remove it, but USB may not detected when plug it.